website/source/layouts/_header.haml

```diff
@@ -0,0 +1,25 @@
+.diamonds
+  .diamonds--container
+    .diamonds--item
+    .diamonds--item
+      = link_to "Blog", "blog.html", class: "diamonds--item-link"
+    .diamonds--item
+    .diamonds--item
+    .diamonds--item
+      .diamonds--item-version-title Version
+      = link_to "Open Changelog", "changelog.html", class: "diamonds--item-changelog"
+      .diamonds--item-version= livingstyleguide_gem_version
+    - 3.times do
+      .diamonds--item
+    %a.diamonds--item(href="/")
+    - 3.times do
+      .diamonds--item
+    %a.diamonds--item(href="/")
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))